### DISCUSSION TITLE
Fast FFT is bomb!

### DISCUSSION THREAD

**Discussion by fagci** [AlexandreRouma/SDRPlusPlus#562] [Show and tell]
![image](https://user-images.githubusercontent.com/8795970/147489539-fc425798-e433-416a-819a-6491da1fd742.png)
Below is ON

![image](https://user-images.githubusercontent.com/8795970/147489663-426c9728-eee8-4505-ab7b-63b39188ff2e.png)

**Comment by AlexandreRouma** [comment]
Although it looks like it, it doesn't reduce noise in the signal, it only makes the FFT less precise but more CPU efficient.
I don't recommend using it if you don't have CPU usage issues.

**Reply by fagci** [reply]
Anyways, that option makes navigation easier, while using plastic RTL-SDR. Thank you for software =)
